docs/resources/domain_records.md

```diff
@@ -60,3 +60,5 @@ resource "namecheap_domain_records" "my-domain2-com" {
 - `type` - (Required) Possible values: A, AAAA, ALIAS, CAA, CNAME, MX, MXE, NS, TXT, URL, URL301, FRAME
 - `mx_pref` - (Optional) MX preference for host. Applicable for MX records only
 - `ttl` - (Optional) Time to live for all record types. Possible values: any value between 60 to 60000
+
+~> It is strongly recommended to set `address` and `hostname` in lower case to prevent undefined behavior!  
```

Comment:
Can we always convert them to lowercase automatically during comparison?

Reply:
Basically, yes.
However, the existing behavior works in the way where if a user changes the camel case of the address of any record, the provider will "re-create" the record. I will have to make the READ smarter.

I will revert on this in the context of a current PR.